code/datums/components/fishing_spot.dm

```diff
@@ -13,7 +13,6 @@
 	else
 		return COMPONENT_INCOMPATIBLE
 	fish_source.on_fishing_spot_init(src)
-	RegisterSignal(parent, COMSIG_ATOM_ATTACKBY, PROC_REF(handle_attackby))
```

Comment:
is this line removal intentional? given the proc `handle_attackby(...)` still exists further down, and I cannot find another place it is called

Reply:
Oh, I forgot to remove the proc as well. It's kinda redundant now since the 'handle cast' signal now handles both melee and ranged.